freshdesk/v2/test.py

```diff
@@ -110,73 +116,31 @@ def _delete(self, url, *args, **kwargs):
         raise HTTPError('404: mocked_api_delete() has no pattern for \'{}\''.format(url))    
 
 
-class TestAPIClass(TestCase):
+class TestAPIClass:
```

Comment:
Pytest's `parametrize` doesn't work with `unittest` test cases, so I had to drop it here.

Reply:
Fine with me, I don't use any base classes with pytest either. Can you remove the unused `unittest` import though?

Reply:
`unittest` was still used by every other test class but I refactored them as well now to be consistent.